### ISSUE TITLE
Does this support PDB Version 2.0?

### ISSUE THREAD

**Issue by MasonLeeBack** [wbenny/pdbex#5]
I'm trying to use the ntoskrnl.pdb from Windows 2000 to extract some structures, but it doesn't seem to know the architecture and returns no structures back.

**Comment by wbenny** [comment]
Can you share the pdb?

**Comment by MasonLeeBack** [comment]
https://drive.google.com/file/d/1NiMU1B5cMapB246DkVEH1S7NpYVXenQu/view?usp=sharing
https://drive.google.com/file/d/1BV3XhD9QpnWKYFBlRWUkvAupBuNIhEm4/view?usp=sharing

One is the executable, one is the pdb

**Comment by wbenny** [comment]
Yep, looks like msdia doesn't understand this old format, unfortunatelly. Custom PDB parser implementation would be needed for this. I'll gladly accept pull request :)

**Comment by MasonLeeBack** [comment]
I'll see what I can do :)
